lsp-go.el

```diff
@@ -47,8 +47,93 @@ completing function calls."
   :type '(repeat string)
   :group 'lsp-gopls)
 
+(defcustom lsp-gopls-build-flags '("-tags")
+  "A set of flags passed on to the build system when invoked,
+  applied to queries like `go list'."
+  :type '(repeat string)
+  :group 'lsp-gopls
+  :risky t
+  :package-version '(lsp-mode "6.3"))
```

Comment:
Untill 6.2 is released we must keep 6.2 versin.

Reply:
Can do :+1: